resources/scss/components/_content.scss

```diff
@@ -1,8 +1,3 @@
-// Contact footer content
-footer .content a {
```

Comment:
This specificity was getting in the way. Luckily I had a solution implemented already (.white-links in global.scss) that I used for the contact footer and removed this code.